apps/explorer/src/components/HomeMetrics/index.tsx

```diff
@@ -22,13 +22,32 @@ function StatsWrapper(props: StatsProps) {
 
 function FormattedStatsAmount({
     amount,
+    truncate,
+    unit,
     ...props
-}: Omit<StatsProps, 'children'> & { amount?: string | number | bigint }) {
+}: Omit<StatsProps, 'children'> & {
+    amount?: string | number | bigint;
+    truncate?: boolean;
+    unit?: string;
+}) {
     const [formattedAmount, postfix] = formatAmountParts(amount);
 
+    let renderAmount = formattedAmount;
```

Comment:
Instead of doing any of this, I'd just do `const numberFormatter = new Intl.NumberFormat('en');` and then `numberFormatter.format(amount)` whenver it's passed in. This is more complicated than it should be.

Reply:
`numberFormatter.format` can only take in a `number` type. Our `amount` can be of `string | number | bigint`. Calling `toLocaleString` works with all of them though however.

Reply:
I think it technically works actually, and you could also do `String(bigint)`, but `toLocaleString` also works